ksql-functional-tests/src/test/resources/query-validation-tests/round-floor.json

```diff
@@ -0,0 +1,82 @@
+{
+  "comments": [
+    "Tests covering the use of the ROUND(DOUBLE), ROUND(DOUBLE,INT), FLOOR(DOUBLE), FLOOR(DOUBLE,INT) function."
+  ],
+  "tests": [
+    {
+      "name": "calculate ROUND function",
+      "statements": [
+        "CREATE STREAM test (v DOUBLE) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT ROUND(v) as R0, ROUND(v, 0) as R00, ROUND(v, 1) as R1, ROUND(v, 2) as R2, ROUND(v, 10) as R10 FROM test;"
```

Comment:
some impls of `ROUND` support negative numbers of the second argument, e.g.

`ROUND(124, -1)` would give `120`.
`ROUND(124, -2)` would give `100`.

Might be worth adding support for this.